integration-tests/itest/deploy_test.go

```diff
@@ -201,6 +207,58 @@ var _ = Describe("Application deployment", func() {
 		})
 	})
 
+	Context("Application swiss-knife is deployed and running", func() {
+
+		It("should TRY reconfigure spark streamlets to add a pvc but find there's no pvc in the cluster", func(done Done) {
+			err := cli.Configure(swissKnifeApp, UpdateMountingPVC)
+			Expect(err).To(HaveOccurred())
+			By("Wait for the deployment of the new configuration")
+			time.Sleep(deploySleepTime) // this wait is to let the application go into deployment
+			cli.PollUntilAppStatusIs(swissKnifeApp, "Running")
+			close(done)
+		}, LongTimeout)
+
+		It("should deploy a pvc", func() {
+			_, err := k8s_pvc.CreatePVC(PVCResourceFile, swissKnifeApp.Name, clientset)
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("should reconfigure spark streamlets to add a pvc and mount it", func(done Done) {
+			err := cli.Configure(swissKnifeApp, UpdateMountingPVC)
+			Expect(err).NotTo(HaveOccurred())
+			By("Wait for the deployment of the new configuration")
+			time.Sleep(deploySleepTime) // this wait is to let the application go into deployment
+			cli.PollUntilAppStatusIs(swissKnifeApp, "Running")
+			close(done)
+		}, LongTimeout)
+
+		It("should write specific content in any akka streamlet", func(done Done) {
+			err := k8s.CopyFile(swissKnifeApp.Name, clientset, "akka", PVCResourceLocalFileMountPath, PVCResourceAkkaFileMountPath)
+			Expect(err).NotTo(HaveOccurred())
+			close(done)
+		}, LongTimeout)
+
+		It("should find specific content in any spark streamlet", func(done Done) {
+			out, err := k8s.ReadFile(swissKnifeApp.Name, clientset, "spark-process", PVCResourceSparkFileMountPath)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(out).To(Equal(PVCResourceFileContent))
+			close(done)
+		}, LongTimeout)
+
+		It("should find specific content in any flink streamlet", func(done Done) {
+			out, err := k8s.ReadFile(swissKnifeApp.Name, clientset, "flink-process", PVCResourceFlinkFileMountPath)
+			Expect(err).NotTo(HaveOccurred())
+			Expect(out).To(Equal(PVCResourceFileContent))
+			close(done)
+		}, LongTimeout)
+
+		It("should delete that pvc", func() {
```

Comment:
Should try delete pvc as well before creating it, if it exists, in case there were left-overs from a previous run.

Reply:
I took care of that at the beginning of the test suite